simulator/infra/cache/cache_tag_array.h

```diff
@@ -1,82 +1,58 @@
 /**
- * cache_tag_array.h
- * Header for the cache tag array model.
- * MIPT-MIPS Assignment 5.
- * Ladin Oleg.
+ * CacheTagArray
+ * @author Oleg Ladin, Denis Los
  */
 
-/* Protection from multi-including. */
 #ifndef CACHE_TAG_ARRAY_H
 #define CACHE_TAG_ARRAY_H
 
-/* C++ libraries. */
-#include <list>
-#include <vector>
 
-/* Simulator modules. */
+#include <infra/cache/LRUTagCache.h>
 #include <infra/types.h>
 #include <infra/log.h>
 
-/* Replacement algorithm modules (LRU). */
-class LRUInfo
```

Comment:
LRU was intentionally put to the separate class as there are a lot of alternative policies (optimal replacement, pseudo-LRU, MRU, weighted-LRU etc.). So, please put the separate class back, later we will introduce the polymorphic replacement policy selection.